wordpress-mobile/WordPress-iOS
# Show a list of installed plugins for Jetpack sites

First part of #7471:

- Feature flag: plugins only enabled on developer builds for now.
- Blog support flag: plugins only enabled for Jetpack sites via REST.
- UI: a basic plugin list with name, state, and a placeholder icon.
- Backend: Implemented remote and unit tests to load plugin list from the API.

![simulator screen shot 8 aug 2017 11 12 23](https://user-images.githubusercontent.com/8739/29064924-d3310e58-7c2a-11e7-89f4-052f624508b1.png)


To test:

- For a Jetpack site, go to Site > Plugins
- Verify that it list the currently installed plugins


Needs review: @elibud
